### ISSUE TITLE
actions.delete_mark can't map key

### ISSUE THREAD

**Issue by PhoenixPtt** [nvim-telescope/telescope.nvim#3115]
### Description

expect to map a key to action of delete mark, but failed

### Neovim version

```markdown
nvim 0.9.5
```

### Operating system and version

ubuntu22.04

### Telescope version / branch / rev

telescope 0.1.6

### checkhealth telescope

```markdown
heathy
```

### Steps to reproduce

1. config the keymap
2.  reopen nvim

### Expected behavior

map key success

### Actual behavior

report error like this:
![image](https://github.com/nvim-telescope/telescope.nvim/assets/68508116/23726a3d-7419-4de1-8c9d-6154ec2aa2b2)

### Minimal config

```Lua
mappings = {
					i = {
						["<C-m>"] = actions.delete_mark,
					},
					n = {
					},
},
```

**Comment by sineptic** [comment]
There are no delete_mark action. 
``` lua
:h telescope.actions -- to see all actions
```

**Comment by PhoenixPtt** [comment]
> There are no delete_mark action. 
> ``` lua
> :h telescope.actions -- to see all actions
> ```

I know why, I use the 0.1.x branch, but I read the document of the master branch. the action of delete_marks exists in the master branch but not the 0.1.x branch. 
Thank you for you reply!

**Comment by PhoenixPtt** [comment]
I use the master branch, and mapping dm to action delete_marks, then I can delete marks successfully. But when close nvim and open it again, I find the marks that I think I have deleted are still there, Why?

**Comment by jamestrew** [comment]
Some marks have special meaning and they're automatically set by vim. There's not much point to deleting them.

The `m{a-zA-Z}` marks are generally the ones you set manually and therefore have most control over deleting them and it staying delete (until you set them again).

**Comment by PhoenixPtt** [comment]
> Some marks have special meaning and they're automatically set by vim. There's not much point to deleting them.
> 
> The `m{a-zA-Z}` marks are generally the ones you set manually and therefore have most control over deleting them and it staying delete (until you set them again).

ok,thank you for your reply!

**Comment by jamestrew** [comment]
@arilebedey it's not available on telescope v0.1.x as it's considered a new feature and we don't merge new features into existing minor release according to semver.
